WalletWasabi.Fluent/Views/Wallets/Buy/UserInputValidatorView.axaml

```diff
@@ -157,47 +185,46 @@
 
       <!-- SupportChatStep -->
       <DataTemplate DataType="vm:SupportChatStep">
-        <TextBox VerticalAlignment="Center"
-                 Text="{Binding Value}"
-                 x:Name="TextBox"
-                 MaxLength="5000"
-                 Watermark="{Binding Watermark}">
-          <TextBox.InnerRightContent>
-            <StackPanel VerticalAlignment="Center" Margin="0 0 15 0" Orientation="Horizontal" Spacing="2">
-              <TextBlock Classes="fade" Text="{Binding #TextBox.Text.Length, FallbackValue=0}" />
-              <TextBlock Classes="fade" Text="/" />
-              <TextBlock Classes="fade" Text="{Binding #TextBox.MaxLength}" />
-            </StackPanel>
-          </TextBox.InnerRightContent>
+        <ChatTextBox Text="{Binding Value}"
+                 Watermark="{Binding Watermark}"
+                 x:Name="TextBox">
+          <ChatTextBox.InnerRightContent>
+            <DockPanel Classes="limit">
+              <StackPanel>
+                <TextBlock Classes="fade" Text="{Binding #TextBox.Text.Length, FallbackValue=0}" />
+                <TextBlock Classes="fade" Text="/" />
+                <TextBlock Classes="fade" Text="{Binding #TextBox.MaxLength}" />
+              </StackPanel>
+            </DockPanel>
+          </ChatTextBox.InnerRightContent>
           <Interaction.Behaviors>
             <FocusOnAttachedBehavior />
+            <TextBoxAddLineBreakBehavior />
```

Comment:
This is not needed since ChatTextBox is used, right?

Reply:
Removed ;)